prac_05/wimbledon.py

```diff
@@ -0,0 +1,53 @@
+"""
+Word Occurrences
+Estimate: 120 minutes
+Actual:   234 minutes
+"""
+
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    """Read data from a csv file then print each name, their number of wins and countries"""
+    data = retrieve_records(FILENAME)
+    championships_to_count, countries = process_records(data)
+    display_records(championships_to_count, countries)
+
+
+def retrieve_records(filename):
+    """Append a record for each line in the file"""
+    records = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()
```

Comment:
Reading the file content can be improved by using CSV reader. Doing it this way is fine in this case since it does not matter if those scores are split up because we do not need to access the scores, but in case if we need the scores, CSV reader will recognize all scores as one entry.